dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/ajax/CMSMaintenanceAjax.java

```diff
@@ -247,6 +247,7 @@ public String deleteContentletsFromIdList(String List, String userId) throws Por
 
 		if (!contentlets.isEmpty()) {
 			for (Contentlet contentlet : contentlets) {
+				contentlet.getMap().put(Contentlet.DONT_VALIDATE_ME, true);

```

Comment:
Should we do this in the API to ensure the delete works everywhere?

Reply:
respecting permissions, of course

Reply:
@wezell one question is this correct: https://github.com/dotCMS/core/blob/master/dotCMS/src/main/java/com/dotmarketing/business/VersionableAPIImpl.java#L362 seems that if you try to unpublish a content that has an expiration date should throw an error. I think is someone wants to unpublish content, they can do it regardless if it has an expiration date.